unit-tests/func/test-set-option.py

```diff
@@ -0,0 +1,102 @@
+import platform
+import pyrealsense2 as rs, test
+import time
+
+#L515
+
+dev = test.find_first_device_or_exit()
```

Comment:
Use L500 product family unless this test should run on D400 -- check profiles and use one that's common to both?

Reply:
done